pkg/metrics/metrics.go

```diff
@@ -0,0 +1,15 @@
+package metrics
+
+import (
+	"github.com/libopenstorage/stork/pkg/extender"
+	"github.com/libopenstorage/stork/pkg/monitor"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// Register stork metrics
+func init() {
+	prometheus.MustRegister(extender.HyperConvergedPodsCounter)
```

Comment:
These should be in `pkg/extender/extender.go` since it is being populated by that package